src/main/java/org/datadog/jenkins/plugins/datadog/DatadogUtilities.java

```diff
@@ -138,8 +134,8 @@ public static Map<String, Set<String>> getBuildTags(Run run, EnvVars envVars) {
         String jobName;
         try {
             jobName = run.getParent().getFullName();
-        } catch (NullPointerException e) {
-            // It can only throw a NullPointerException when running tests
+        } catch (Exception e) {
```

Comment:
Looks like we can remove this `catch` altogether: `run` is never `null` (since there is only one usage of this method in the plugin, and run is checked for null earlier in the caller's code), and `run.getParent()` also never returns `null` (since that method is annotated with `@NonNull`)

Reply:
Done.